Buildings/Controls/OBC/ASHRAE/G36/AHUs/SingleZone/SetPoints/ReliefDamper.mo

```diff
@@ -57,21 +51,19 @@ protected
     annotation (Placement(transformation(extent={{-20,70},{0,90}})));
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant minRelDam(
     final k=minRelDamPos)
-    "The relief damper position that maintains a building pressure of 0.05 inchWC (12.44 Pa) while the system is at MinPosMin 
-    (i.e., the economizer damper is positioned to provide MinOA while the supply fan is at minimum speed)."
+    "Relief damper position maintaining building static pressure at setpoint while the system is at MinPosMin"
     annotation (Placement(transformation(extent={{-80,-50},{-60,-30}})));
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant maxRelDam(
     final k=maxRelDamPos)
-    "The relief damper position that maintains a building pressure of 0.05 inchWC (12.44 Pa) while the economizer damper is 
-    fully open (physical maximum) and the fan speed is at cooling maximum."
+    "Relief damper position maintaining building static pressure at setpoint while while the system is at MaxPosMax"
     annotation (Placement(transformation(extent={{-80,-90},{-60,-70}})));
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant minPosAtMinSpd(
     final k=minPosMin)
     "Outdoor air damper position, when fan operating at minimum speed to supply minimum outdoor air flow"
     annotation (Placement(transformation(extent={{-40,-30},{-20,-10}})));
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant outDamPhyPosMaxSig(
     final k=outDamPhyPosMax)
-    "Physical or at the comissioning fixed maximum position of the economizer damper - economizer damper fully open. "
+    "Physical or at the comissioning fixed maximum position of the economizer damper"
```

Comment:
Please use outdoor air damper instead (since all 3 are economizer dampers in essence).